.github/workflows/complete-build.yml

```
@@ -306,33 +306,43 @@ jobs:
+
       - name: Checkout repository
+        if: ${{ matrix.if_condition }}
         uses: actions/checkout@v4
 
       - name: Setup Node.js with Corepack
+        if: ${{ matrix.if_condition }}
         uses: actions/setup-node@v4
         with:
           node-version: ${{ env.NODE_VERSION }}
           cache: 'yarn'
           corepack: true
```

actionlint (1.7.9)

input "corepack" is not defined in action "actions/setup-node@v4". available inputs are "always-auth", "architecture", "cache", "cache-dependency-path", "check-latest", "mirror", "mirror-token", "node-version", "node-version-file", "registry-url", "scope", "token"

(action)